Some fixes

dlorente opened the pull request:
Murilo,

Fiz umas alterações. Adicionei o CSRF e corrigi um erro no deplay do fabfile.

dlorente committed:
Add CSRF protection

dlorente committed:
Create uwsgi log's folder to fix fabfile deploy error

murilobsd commented:
Opa @dlorente valeuz! Eu irei dar uma olhada nas alterações e já te falo!

murilobsd commented on zeusproject/commands.py:
```diff
@@ -414,6 +414,15 @@ def _uwsgi(self):
 
+    def _uwsgi_log_folder(self):
+        uwsgi_log_folder = os.path.join(self.projectfolder, 'log')
+        
```
@dlorente faça uma checagem antes de dar o try, caso o usuário já tenha o diretório log dentro do projeto sempre irá falhar! Outra coisa não use nesse caso em específico o makedirs porque caso altere o nome do projeto irá criar toda a pasta anterior... use o os.makedir() e sempre analisar a excessao nesse caso se usar o os.makedir(path) caso já exista irá retornar FileExistsError:

```python
# se existe o diretorio para que recriar
if os.path.exists(uwsgi_log_folder):
    return

try:
    os.makedir(uwsgi_log_folder)
except FileExistsError:
    raise CreateFolderException("Log path: [%s] already exists." % uwsgi_log_folder)
except Exception:
    raise CreateFolderException("Error creating uwsgi log folder.")
```

_sao duas alternativas_

dlorente replied:
Feito!

dlorente replied:
@murilobsd, fiz uns testes. Parece estar funcionando.

murilobsd commented:
Fiz uma observação, quanto ao csrf está ok! Só faça a checagem do diretório que eu aceito o pull, mais uma vez obrigado.!

dlorente committed:
Changing from os.makedir to os.mkdir.

This prevent the creation of unwanted folders in case of change in the project name
